tests/behat/theme_boost_union_general.feature

```diff
@@ -70,6 +70,44 @@ Feature: Configuring the theme_boost_union plugin as admin
     And ".admin_settingspage_tabs_with_tertiary" "css_element" should not exist
     And I should see "Category: Boost Union" in the "#region-main h2" "css_element"
 
+  @javascript
+  Scenario: Show inactive-theme warning on settings overview and all Boost Union sub pages
+    Given I log in as "admin"
+    And I navigate to "Appearance > Themes" in site administration
+    # Ensure Boost Union is NOT active (switch to Boost).
+    And I click on "Select theme" "button" in the "#theme-card-boost" "css_element"
+    # Overview page should show the warning.
+    When I navigate to "Appearance > Boost Union > Settings overview" in site administration
+    Then I should see "Boost Union is currently NOT the active theme" in the ".alert-warning" "css_element"
```

Comment:
This Behat scenario is correct and working.
If you want to make it even better, you can add a step like
`Then "Boost Union is currently NOT the active theme" "text" should appear before ".admin_settingspage_tabs_with_tertiary" "css_element"`
to each settings page to make sure that the notification is not just shown _anywhere_ but where you want it to appear.